CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v3.1.0 / 2025-04-28
+
+- [IMPROVEMENT] Add `tokenFilePath` support for providing SASL OAuth token via file.
```

Comment:
```suggestion
- [IMPROVEMENT] Add `tokenFilepath` support for providing SASL OAuth token via file.
```

Luckily `tokenFilepath` was actually used, as this is consistent with all the other `....Filepath` configs.